lition/README.md

```diff
@@ -0,0 +1,28 @@
+# Chainlink External Adapter for Lition
+
```

Comment:
Please document what is this adapter used for. I can't figure out what are we pricing here. The price `"price": 34.07` is a price of what exactly?

Reply:
Good point. Added to README.